docs/how-to/enable_hugepages_for_upf.md

```diff
@@ -0,0 +1,68 @@
+# Enable HugePages for UPF
+
+## Requirements
+
+- A Kubernetes host with a CPU supporting AVX2 and RDRAND instructions (Intel Haswell, AMD Excavator or equivalent)
+- A Kubernetes cluster with the Multus addon enabled
+- Juju = 3.1/stable
+- HugePages is enabled in Kubernetes Host and it has enough 2Gi free HugePages 
+- Kubernetes Host CPU should have the flag "pdpe1gb" which allows to set 1Gi HugePages
+
+## Deploy the UPF
+
+Deploy the UPF charm in a Juju model named `user-plane`:
+
+```bash
+juju add-model user-plane
+juju deploy sdcore-upf --trust --channel=edge
+```
+
+## Enable HugePages
+
+```bash
+juju config sdcore-upf enable-hugepages=True
+```
+
+Container resources are hardcoded with 2 CPU and HugePage resources (requests:hugepages-1Gi, limits:2Gi).
+
+```yaml
+limits:
+  cpu: "2"
+  hugepages-1Gi: 2Gi
+requests:
+  cpu: "2"
+  hugepages-1Gi: 2Gi
+```
+
+Check the bessd container logs which indicates that hugepages are reserved:
```

Comment:
```suggestion
Read the bessd container logs which indicates that HugePages are reserved:
```

Reply:
hugepages is being replaced with HugePages. However we are looking for something in the logs. "Check" seems more appropriate here.